docs/source/example_overview.md

```diff
@@ -0,0 +1,76 @@
+# Examples
+
+_The best place to learn about examples in TRL is our [docs page](https://huggingface.co/docs/trl/index)!_
+
+## Introduction
+
+The examples should work in any of the following settings (with the same script):
+   - single CPU or single GPU
+   - multi GPUS (using PyTorch distributed mode)
+   - multi GPUS (using DeepSpeed ZeRO-Offload stages 1 & 2)
+   - fp16 (mixed-precision) or fp32 (normal precision)
+
+To run it in each of these various modes, first initialize the accelerate
+configuration with `accelerate config`
+
+**NOTE for to train with a 8-bit model a more recent version of**
+transformers is required, for example:
```

Comment:
The formatting of this is a bit odd. Maybe also mention 4-bit. Also the installation command actually installs bnb and peft and not transformers?